magenta/models/music_vae/data.py

```diff
@@ -242,8 +241,11 @@ def _maybe_sample_outputs(self, outputs):
   def to_tensors(self, item):
     """Python method that converts `item` into list of tensors."""
     results = self._to_tensors(item)
-    sampled_results = self._maybe_sample_outputs(zip(*results))
-    return list(zip(*sampled_results)) if sampled_results else ([], [])
+    if self.max_tensors_per_item:
```

Comment:
Also make this change in the NoteSequence one. Bonus points if you can use shared code, e.g., by calling this super-method.

Reply:
Done.